package.json

```diff
@@ -7,37 +7,51 @@
         "url": "https://github.com/Telefonica/mistica-web"
     },
     "homepage": "https://github.com/Telefonica/mistica-web",
-    "main": "dist/index.js",
-    "module": "dist-es/index.js",
     "files": [
-        "dist/**",
-        "dist-es/**",
-        "css/**",
-        "community.d.ts",
-        "community.js"
+        "dist/**"
     ],
+    "type": "module",
+    "exports": {
+        "./dist/*": {
+            "default": null
+        },
+        ".": {
+            "types": "./dist/index.d.ts",
+            "import": "./dist/index.js",
+            "default": "./dist/index.js"
+        },
+        "./community": {
+            "types": "./dist/community/index.d.ts",
+            "import": "./dist/community/index.js",
+            "default": "./dist/community/index.js"
+        },
+        "./css/*.css": {
+            "import": "./dist/css/*.css",
+            "default": "./dist/css/*.css"
+        }
+    },
     "scripts": {
-        "test": "jest --config jest.unit.config.js",
-        "test-acceptance": "test-acceptance",
-        "test-ssr": "test-acceptance --config jest.ssr.config.js",
+        "test": "jest --config jest.unit.config.cjs",
+        "test-acceptance": "JEST_PUPPETEER_CONFIG=jest-puppeteer.config.cjs test-acceptance --config jest.acceptance.config.cjs",
+        "test-ssr": "JEST_PUPPETEER_CONFIG=jest-puppeteer.config.cjs test-acceptance --config jest.ssr.config.cjs",
```

Comment:
The `JEST_PUPPETEER_CONFIG` env var tells `jest-environment-puppeteer` where to find the `puppeteer` config.
This is needed because we're using an old version of the `jest-environment-puppeteer` library (v6) and it is unaware of `.cjs` files. Newer versions don't have this issue but we cannot upgrade because this requires a `puppeteer` upgrade.